.circleci/config.yml

```diff
@@ -0,0 +1,41 @@
+version: 2
+
+checkout:
+  post:
+    - >
+      if [ -n "$CI_PULL_REQUEST" ]; then
+        PR_ID=${CI_PULL_REQUEST##*/}
+        git fetch origin +refs/pull/$PR_ID/merge:
+        git checkout -qf FETCH_HEAD
+      fi
+
+defaults:
+  - &test_defaults
+    docker:
+      - image: circleci/python:2.7
+    steps:
+      - checkout
+      - run:
+          name: Install tox
+          command: sudo pip install tox
+      - run:
+          name: Run tox of specfic environment
+          command: tox -e $DO_ENV
+jobs:
+  flake8:
+    <<: *test_defaults
+    environment:
+      DO_ENV: flake8
+
+  docs:
+    <<: *test_defaults
+    environment:
+      DO_ENV: docs
+
+workflows:
+  version: 2
+
+  build_and_test:
+    jobs:
+      - flake8
+      - docs
```

Comment:
remove the docs
what about py27 and py26 tests

Reply:
There no tests